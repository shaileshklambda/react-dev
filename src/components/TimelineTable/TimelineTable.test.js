import React from "react";
import renderer from "react-test-renderer";
import Timelinetable from ".";

it("Timelinetable: default", () => {
  const component = renderer.create(<Timelinetable />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
