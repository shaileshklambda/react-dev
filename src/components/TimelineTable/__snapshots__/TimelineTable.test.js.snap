// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timelinetable: default 1`] = `
<div>
  No Data to display
</div>
`;
